src/tool_getparam.c

```diff
@@ -999,6 +1004,76 @@ static CURLcode set_trace_config(struct GlobalConfig *global,
   return result;
 }
 
+#ifdef HAVE_SETSOCKOPT_SOL_IP
```

Comment:
The command line option parsing cannot depend on this. If it should, it needs to check if *this* libcurl supports it in run-time.

Reply:
The problem is that all the "user-friendly" values here are defined in `netinet/ip.h`, which is not available on all platforms. What do you suggest to do? Maybe support only numeric values on other platforms? But what for actually, if it has no effect?

Reply:
We work hard to decouple the tool from the library and its specific versions and features. What if you run with a different build of the library - with the same tool version, how would the tool know what features the library supports?

Reply:
If you prefer that, I'll just remove the readable names, and let it just accept a numeric value (that's what we use anyway. The idea to use names was taken from #7762). Then I can remove the condition.

Reply:
By the way, `ECH` is also disabled in command-line with `#ifdef`. Is that wrong too?

Reply:
Done, I hope you like it now :)